docs/pages/components/select.md

```diff
@@ -23,24 +23,24 @@ For lists that require more than 12 options, the <a href="/patterns/combobox-inp
 <div class="fd-popover">
   <div class="fd-popover__control">
      <div class="fd-select">
-         <div class="fd-select__control" tabindex="0" aria-controls="h0C6A325" aria-expanded="false" aria-haspopup="true">
+         <div class="fd-select__control" role="button" tabindex="0" aria-controls="h0C6A325" aria-expanded="false" aria-haspopup="true">
              Select
```

Comment:
Should this `div` be a button instead? `Enter` or `Space` don't seem to open the selection list box. Or would that be implemented by consumers?

Reply:
I initially tried changing it to a `button`, but there were a lot of style changes that would need to be overridden and I couldn't easily get the text to be centered. I believe leaving it to the implementation libraries should be fine, as the docs are mainly for showing styles and not functionality. I'm not sure which file these keyboard events would be added if they had to be handled.

Reply:
So this would be a button in the implementation libraries but a div here? That's not generally the pattern we've followed in the past

Reply:
No, this is exactly the markup the implementation libraries would follow. I meant that they would need to implement Enter and Space handling in the JavaScript.

Reply:
Thanks for clarifying!